src/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -655,6 +657,44 @@ ${ flowData.code }
 
 	}
 
+	getDirective( name, shaderStage = this.shaderStage ) {
```

Comment:
Imo, this method name is confusing since it returns nothing. A `get*()` method should do that however.

I suggest you rename the method to `enableDirective()`.

Reply:
Certainly. Apologies for these minor but consistent errors, I'm working to become more professional with how I write and present my code 👍